Islandora/islandora_defaults
# Newspaper

**GitHub Issue**: (link)
* https://github.com/Islandora/documentation/issues/1356

# What does this Pull Request do?
I've refactored to package the newspaper functionality for islandora from install profile.

The main logic is packaged into its own module: https://github.com/Natkeeran/views_nested_details.  

Almost as it is from the install profile except for minor default css for the accordion.

The view and context for block placement is packaged into a small sub module in this PR.

There is a known issue with Month display in the EDTF formatter (1970-February vs  February).

# How should this be tested?
* Have a standard installation (i.e https://github.com/Islandora-Devops/islandora-playbook)
* Have some newspaper content (i.e https://github.com/digitalutsc/repo_islandora_demo_objects - https://docs.google.com/spreadsheets/d/1DV0Ka0ZafZq3RgCn0x_AetFtlmnMuNFltzck2QL0ApY/edit#gid=994664908)
* Get the dev branch of the controlled access terms - https://github.com/Islandora/controlled_access_terms
* Install https://github.com/Natkeeran/views_nested_details
* Enable Islandora Newspaper
* Visit a newspaper item.  It should contain a block with an accordion view that will enable you to navigate the newspaper issues.
